### PR TITLE
UP-2805 - Adds github actions

### PR DESCRIPTION
**Pull request by gustavoaz7**

### CODE REVIEW

**Review by uschischroder — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+.PHONY: build
+build: node_modules
+	npm run build
+
+.PHONY: lint
+lint: node_modules
+	npm run lint
+
+.PHONY: ci
+ci: lint build
```
Comment (enjoythelive1) [inline Makefile:30]:
```suggestion
.PHONY: node_modules
node_modules:
	npm i

.PHONY: build
build:
	npm run build

.PHONY: lint
lint:
	npm run lint

.PHONY: ci
ci: node_modules lint build
```

**Review by enjoythelive1 — CHANGES_REQUESTED** [review]
The CI should install the dependencies using `npm ci`

**Comment by Drafter500** [comment]
The actions have not been run in this branch, do we miss something here?

**Comment by gustavoaz7** [comment]
> The actions have not been run in this branch, do we miss something here?

They have. Here is one example https://github.com/35up/RapiDoc/actions/runs/1998585647
I guess latest commit did not due to late github issues as victor mentioned in the standup

**Review by enjoythelive1 — APPROVED** [review]

**Review by Drafter500 — APPROVED** [review]

**Pull request merged**